3570/3/rating_places/db/migrate/20190729141144_update_column_comments_count.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateColumnCommentsCount < ActiveRecord::Migration
+  def change
```

Comment:
You are the only 1 who contribute to this app. Don't create those migrations, please. Better modify migration that contains this column. In this case, it's `20190727123543_create_places.rb`